x-pack/plugins/fleet/server/types/models/preconfiguration.ts

```diff
@@ -77,5 +85,8 @@ export const PreconfiguredAgentPoliciesSchema = schema.arrayOf(
         ),
       })
     ),
-  })
+  }),
+  {
+    defaultValue: [DEFAULT_AGENT_POLICY, DEFAULT_FLEET_SERVER_AGENT_POLICY],
```

Comment:
what happens if the user has only specified _either_ a default or a default fleet server policy in their preconfiguration? will we still ensure that whichever default one is missing, is added? what if their preconfiguration doesn't specify any default policies?

i think we might want to do the same sort of merging downstream like what you did for default packages

Reply:
I was under the impression from discussion on the issue (https://github.com/elastic/kibana/issues/97016) that we explicitly did not want to do downstream merging for agent policies, and that the config file should be the source of truth for what the "default policies" are.